bluesky/examples.py

```diff
@@ -21,6 +21,7 @@ def describe(self):
 
 class Base:
     def __init__(self, name, fields):
+        self._name = name
```

Comment:
It should be `name`, which is now part of the "bluesky interface" that all ophyd devices and signals are expected to have. Nothing should still be expecting a `_name` attribute.